### PR TITLE
chore(components): fix storybook for IE11

### LINKED ISSUE #357: Storybook doesn't work on IE11
When accessing to talend.surge.sh/components from IE11, the storybook doesn't load. The following error appears in the console :

`SCRIPT1002: Syntax error`

It fails on `module.exports = val => val.replace(ESCAPED_CHARACTERS, '\\$1');`

### PR DESCRIPTION
**Pull request by ncomont**
This fixes #357

**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
The storybook doesn't load on IE11

**What is the new behavior?**
The storybook works on IE11

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://358.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://358.talend.surge.sh)

File: packages/components/.babelrc [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {
   "presets": [
     "es2015",
-    "react"
+    "react",
```
Comment (jmfrancois) [inline packages/components/.babelrc]: are you sure this is working in RC file ?
because AFAIK this is json, and this is not a valid json

Comment (ncomont) [inline packages/components/.babelrc]: I had no problem during my tests, but I'll delete it, thank you.

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://358.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://358.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://358.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://358.talend.surge.sh)

**Pull request merged**